src/SFA.DAS.Assessor.Functions/Startup.cs

```diff
@@ -86,9 +86,7 @@ public override void Configure(IFunctionsHostBuilder builder)
             {
                 var sftpSettings = s.GetService<IOptions<SftpSettings>>()?.Value;
                 return new SftpClient(sftpSettings.RemoteHost, Convert.ToInt32(sftpSettings.Port), sftpSettings.Username, sftpSettings.Password);
-            });
-
-            builder.Services.AddNotificationService();
```

Comment:
Is the Notifications API being removed completely now - if so - have you removed it all?